testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -1,6 +1,9 @@
 # Copyright 2015-2022 SUSE LLC
 # Licensed under the terms of the MIT license.
 
+### This file contains all step definitions concerning salt and bootstrapping
```

Comment:
This change will collide with the other PR.

Reply:
jup seen it, probably cherr-picked wrong